src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -27,6 +29,12 @@ public override void Write(string message)
             {
                 _writer.Write(message);
             }
+
+            public override void WriteLine(string message)
+            {
+                if (ShouldOverrideWriteLine)
+                    _writer.WriteLine(message);
```

Comment:
I wasn't needed, but to test the log file functionality that was added it is needed. Will add it with the other tests.